### PR TITLE
Feature/encapsulation

### PR DESCRIPTION
**Pull request by ProgHunter**
## Description
Ensure all class attributes are private and have accessors if needed.

## Fixes
Link all the related issues from the issue tracker
- Closes: #31 

## How has this been tested ?
- Tested locally by building and running it

## Checklist
- [x] My code follows the style guidelines of this project
- [x] I have performed a self-review of my own code
- [x] I have commented my code, particularly in hard-to-understand areas
- [x] I have made corresponding changes to the documentation
- [x] My changes generate no new warnings

### CODE REVIEW

File: src/proc_image_processing/algorithm/general_function.h [hunk 1]
```diff
@@ -74,8 +74,7 @@ namespace proc_image_processing {
   // total number of pixel in the upright rectangle
   float CalculatePourcentFilled(const cv::Mat& image, const cv::Rect& rectangle);
 
-  float CalculatePourcentFilled(const cv::Mat& image,
```
Comment (wtang-8) [inline src/proc_image_processing/algorithm/general_function.h:77]: Renommer la fonction: CalculatePercentFilled.

File: src/proc_image_processing/filters/detectors/pipe_angle_detector.h [hunk 2]
```diff
@@ -1,21 +1,6 @@
 /// \author	Pierluc Bédard <pierlucbed@gmail.com>
 /// \author	Jérémie St-Jules Prévôt <jeremie.st.jules.prevost@gmail.com>
```
Comment (wtang-8) [inline src/proc_image_processing/filters/detectors/pipe_angle_detector.h:2]: Enlever les sections \author aussi?

**Review by wtang-8 — DISMISSED** [review]
looks good to me

File: src/proc_image_processing/filters/detectors/pipe_angle_detector.h (hunk 2 above)
Comment (ProgHunter) [reply src/proc_image_processing/filters/detectors/pipe_angle_detector.h:2]: Suivant les exigences de la tâche "NF-1.4 Nettoyage du repo" seul les champs author et date ont été gardés pour l'entête des fichiers.

File: src/proc_image_processing/algorithm/general_function.h (hunk 1 above)
Comment (ProgHunter) [reply src/proc_image_processing/algorithm/general_function.h:77]: Done!

**Review by Ismedrasor — APPROVED** [review]
Modification have been approve, @willDBZ can you close your change request?

**Review by Ismedrasor — APPROVED** [review]

**Pull request merged**